internal/server/utils/redisCmdAdapter.go

```diff
@@ -0,0 +1,72 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/dicedb/dice/internal/cmd"
+)
+
+func ParseHTTPRequest(r *http.Request) (*cmd.RedisCmd, error) {
+	prefix := strings.TrimPrefix(r.URL.Path, "/")
```

Comment:
nit: just for better clarity.
```suggestion
	command := strings.TrimPrefix(r.URL.Path, "/")
```